api/build.gradle

```diff
@@ -18,8 +18,9 @@ dependencies {
     testImplementation project(':grpc-context').sourceSets.test.output,
             project(':grpc-testing'),
             project(':grpc-grpclb'),
-            libraries.guava_testlib
-
+    testImplementation (libraries.guava_testlib) {
```

Comment:
Remove the comma above, otherwise this is considered as appending its return value to the above list, although not breaking.

Reply:
done